docs/general/faq/faq.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_label: '❓ FAQ'
+sidebar_position: 6
+description: 'Frequently asked questions'
+---
+
+# FAQ
+
+## Does LUKSO have a grants programme?
+
+Yes, absolutely! Please go to our main page [here](https://lukso.network/grants) for more information and to apply!
```

Comment:
Suggest not using the word `here` as link, as it is harder to see the link in the text.

```suggestion
Yes, absolutely! Please go to our main [grants program page](https://lukso.network/grants) for more information and to apply!
```